api/src/main/resources/beans_2_0.xsd

```diff
@@ -353,5 +354,14 @@
       </xs:choice>
     </xs:complexType>
   </xs:element>
+  <xs:element name="trim" type="xs:string" fixed="">
```

Comment:
My science in xsd is not very high, but my friend Google told me that: 

> The all element specifies that the child elements can appear in any order and that each child element can occur zero or one time.

(See ref [here](http://www.w3schools.com/xml/el_all.asp))

So `minOccurs` attributes are useless here IMO.